bundles/AdminBundle/templates/admin/asset/show_version_unknown.html.twig

```diff
@@ -30,7 +30,11 @@
 <table id="wrapper" cellpadding="0" cellspacing="0" border="0">
     <tr>
         <td>
-            Sorry, no preview available
+            Sorry, no preview available<br>
+            {% set tempFile = asset.getTemporaryFile() %}
+            {% set dataUri = pimcore_asset_version_preview(tempFile) %}
+
+            <a href="{{ dataUri }}" download="{{ asset.getFilename() }}">{{ 'download'|trans([],'admin') }}</a>
```

Comment:
This is cool, might be quite handy.

Just an input, would it be good to add a prefix (eg. date or version id?) to the filename (under the attribute `download`) to differenciate historical versions?

Reply:
Thanks. I now added the version date as file name suffix, e.g. when asset file name is 123.doc then for a version of date 2023-01-10 the download file name will be 123-2023-01-10.doc.

Reply:
better, but might be a "problem" if one uploaded multiple files on the same date, eg.
![image](https://user-images.githubusercontent.com/6014195/211506996-862d3724-2965-48c1-ada0-4280aa91e289.png)

Reply:
Right, so I added also the timestamp

Reply:
Thank you! 
Just one last thing, the files in the screenshot gets named as `2023-01-10-08-59-26` (1 hour difference), somehow the timezone didn't got considered here

Reply:
This was the UNIX timestamp (which is by definition in GMT). Now I converted it it to the server timezone.
The displayed timezone in the versions panel is a bit different: Here the UNIX timestamp from the version gets transfered and then converted to a human readable date by ExtJS (which also does timezone adjustment).

Reply:
Mhhh, still don't match, for example, i haven't changed the server timezone (and by default is UTC), while in frontend ExtJS "automatically" knows it should convert to CET.

Reply:
This is not "automatically", this is because your browser knows which timezone it is in. Of course we could also do the conversion of the UNIX timestamp with JS but is it really worth it?

Reply:
mhhh, what about manage the date suffix in frontend ExtJs, since the purpose is to help end-user to distinguish between files and they likely would expect that the date matches the one that one sees in the versions grid. 

All these conversion could be too troublesome, if server timezone is something UTC+8 and user is browsing from UTC-3, still would end up in a mismatch.

Reply:
Ok, now it uses browser / user timezone.

Reply:
It works!
Maybe a little too vanilla JS, what about the following change?
```
<script>
    document.addEventListener('DOMContentLoaded', function () {

        {% set filenamePrefix = asset.getFilename()|split('.')|slice(0,-1)|join('.') %}
        {% set filenameSuffix = asset.getFilename()|split('.')|last %}
        let versionDate = Ext.Date.format( new Date({{ version.getDate() * 1000 }}), 'Y-m-d-H-i-s');
        document.getElementById('download-version-link').setAttribute('download', "{{ filenamePrefix }}-"+ versionDate+".{{ filenameSuffix }}");
    }, false);
</script>

<!-- library scripts -->
<script src="/bundles/pimcoreadmin/js/../extjs/js/ext-all.js" {{ pimcore_csp.getNonceHtmlAttribute()|raw }}></script>
```

Reply:
To me this would be ok. Others may say that loading complete ExtJS only to format a date would be like taking a sledgehammer to crack a nut ;-)
Have changed it ;-)

Reply:
Thank you!
I had the exact thoughts 😄
In addition, thought that to avoid that, it could be solved by replacing the DOM content instead of using iframes, but that may require too much refactoring for just this minor drawback of loading extjs from cache again

Reply:
With DOM content instead of iframes for version preview I think the PDF preview would not work. But even if it would, let's finish this PR for now and maybe enhance the behaviour later...